earthpy/clip.py

```diff
@@ -4,11 +4,10 @@
 
 
 def _clip_points(shp, clip_obj):
-    """ A function to clip point geometry using geopandas. Takes an
-    input point GeoDataFrame that will be clipped to the clip_obj
-    GeoDataFrame.
+    """Clip point geometry using geopandas.
 
-    Points that intersect with the geometry of clip_obj are extracted
+    Take an input point GeoDataFrame that will be clipped to the clip_obj
+    GeoDataFrame. Points that intersect with the geometry of clip_obj are extracted
     and returned.
 
```

Comment:
how about: 
`Clip an input point GeoDataFrame to the polygon extent of the clip_obj parameter. Points that intersect the clip_obj geometry are extracted with associated attributes and returned.`